apps/frontend/src/layout/AuthLayout.tsx

```diff
@@ -0,0 +1,83 @@
+import { Outlet } from "react-router-dom";
+import AuthBG from "../assets/images/auth-bg.png";
+import Profile1 from "../assets/images/loginScreen/profile-1.png";
+import Profile2 from "../assets/images/loginScreen/profile-2.png";
+import Profile3 from "../assets/images/loginScreen/profile-3.png";
+import Logo from "../assets/images/logo.png";
+
+export function AuthLayout() {
+  return (
+    <main className="flex w-screen h-screen bg-intg-black">
+      <div
+        className="flex w-[45%] flex-col justify-between px-12 pb-8 pt-12"
+        style={{
+          backgroundImage:
+            "radial-gradient(rgba(28, 15, 89, 0.30) 50%, rgba(5, 5, 5, 0.30))",
+          backgroundRepeat: "no-repeat",
+          backgroundPosition: "center",
+          backgroundSize: "cover",
+        }}
+      >
+        <div>
+          <img src={Logo} alt="Logo" />
+        </div>
+
+        <Outlet />
+
+        <div className="self-center max-w-xs text-base text-center text-intg-text">
+          By signing up, you agree to Integraflow Privacy and terms services
```

Comment:
@AfeezGL Can we move this to outlet? This should only appear on the sign up page.